src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1560,10 +1561,24 @@ public ServiceStartupType StartupType
                 startupType = value;
             }
         }
+
         // We set the initial value to an invalid value so that we can
         // distinguish when this is and is not set.
         internal ServiceStartupType startupType = ServiceStartupType.InvalidValue;
 
+        /// <summary>
+        /// The following is the definition of the input parameter "SecurityDescriptorSddl".
+        /// Sets the SecurityDescriptorSddl of the service using a SDDL string.
```

Comment:
The public docs must start with "Gets or sets ...".
I suggest to remove line 1570 at all.

Reply:
Removed